index.html

```diff
@@ -748,7 +733,7 @@ <h6>Контакты</h6>
       <div class="modal-dialog modal-dialog-scrollable">
         <div class="modal-content">
           <div class="modal-header">
-            <h5 class="modal-title" id="exampleModalLabel">Путь Frontend разработчика</h5>
+            <h5 class="modal-title" id="reactExampleModalLabel">Путь Frontend разработчика</h5>
```

Comment:
We must be sure that whether we need to do some changes in `style.css` as well or not

Reply:
Check the style.css whether their is a usage of it there, but I can't find none.